app/views/brexit_landing_page/_page_header.html.erb

```diff
@@ -28,11 +28,9 @@
         } %>
       </div>
       <div class="govuk-grid-column-two-thirds">
-        <%= render "govuk_publishing_components/components/title", {
-          title: I18n.t("brexit_landing_page.page_header"),
-          inverse: true,
-          margin_bottom: 5
-        } %>
+        <h1 class="landing-page__page_header">
+          <%= I18n.t("brexit_landing_page.page_header") %>
+        </h1>
```

Comment:
I wish the component model had been able to accommodate this requirement. Sorry.

Reply:
I started by trying to add `margin-top` with a value in the same way it takes a `margin-bottom`. Though i can see the rational for not giving us a ton of css leavers and buttons. 

I wonder how much of a weird use case is this?

The language toggle comes with a margin-top of `45px` our h1 comes with a `40px` margin-top.